AppCenterCrashes/AppCenterCrashesTests/MSCrashesTestUtil.m

```diff
@@ -47,6 +47,10 @@ + (BOOL)copyFixtureCrashReportWithFileName:(NSString *)filename {
     return NO;
 
   NSError *error = nil;
+  NSString *desPath = [plcrCrashesDir stringByAppendingPathComponent:@"live_report.plcrash"];
```

Comment:
At Xcode 10.2, it couldn't copy the `live_report.plcrash` file into crash cache path: com.plausiblelabs.crashreporter.data when the file with same name exists in the cache path.
It will occur the issue: 
![image](https://user-images.githubusercontent.com/47266368/56334001-234bc600-61c9-11e9-8ad8-5b265e15c013.png)
But at Xcode 9.x, the copy action can work well. 

So I add these codes as the workaround to fix this build error.
Do you have any other ideas for it